packages/security/lib/string.js

```diff
@@ -1,10 +1,8 @@
-const _ = require('lodash');
 const slugify = require('@tryghost/string').slugify;
 
-module.exports.safe = function safe(string, options) {
-    options = options || {};
+module.exports.safe = function safe(string, options = {}) {
     let opts = {requiredChangesOnly: true};
-    if (!_.has(options, 'importing') || !options.importing) {
+    if (!('importing' in options) || !options.importing) {
         opts.requiredChangesOnly = false;
     }
```

Comment:
```suggestion
    const opts = {requiredChangesOnly: options.importing === true};
```


I think this is the same exact logic (since `undefined !== true`), but it's way more readable (not going to lie, took me longer than I would like to admit to understand what the original impl. was doing 😬)